### DISCUSSION TITLE
[Feature] Timeline View in Galleries

### DISCUSSION THREAD

**Discussion by branchmispredictor** [immich-app/immich#2383] [Feature Request]
### The feature

I quite like the timeline view native to the "Photos" section. It would be nice if this view could be reused for looking through images in Galleries too. This would be especially useful in larger galleries.

In terms of implementation - this could also mean unifying the code for the two currently different views with their currently different behaviors.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by tbleiker** [comment]
Another “thumb up” from me.

In addition, I would like to have the timeline view for search results or when showing pictures of a person (face recognition) as well!
